judgels-backends/uriel/uriel-api/src/main/java/judgels/uriel/api/contest/submission/bundle/ContestBundleSubmissionsResponse.java

```diff
@@ -0,0 +1,20 @@
+package judgels.uriel.api.contest.submission.bundle;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import java.util.Map;
+import judgels.jophiel.api.profile.Profile;
+import judgels.persistence.api.Page;
+import judgels.sandalphon.api.submission.BundleSubmission;
+import judgels.uriel.api.contest.submission.ContestSubmissionConfig;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@JsonDeserialize(as = ImmutableContestBundleSubmissionsResponse.class)
+public interface ContestBundleSubmissionsResponse {
+    Page<BundleSubmission> getData();
```

Comment:
This should be the `BundleSubmission` that contains all items (see my other comments)

Reply:
Will be fixed in a new pull request to add bundle submission contest summary/user answers endpoints